src/gg.regression.unity.bots/Runtime/Scripts/RGServiceManager.cs

```diff
@@ -49,7 +40,7 @@ public bool IsAuthed()
             return _rgAuthToken != null;
         }
 
-        public bool LoadAuth()
```

Comment:
Nit: It could be nice to have an overloaded use of this method, to match how it gets used a bunch below. Like:
```
public bool LoadAuth()
{
    return LoadAuth(out _);
}
```